README.md

```diff
@@ -1,6 +1,6 @@
 # Google Student Training in Engineering Program
 
-This repo contains the projects you'll work on for the first half of STEP.
+This repo contains John Dallard's portfolio and STEP projects for his internship.
```

Comment:
This is mostly to test the back-and-forth of the review process, so I'll try to make it (somewhat) meaningful with a trivial change request:

Does this comment need to be in third person? Why not something like this?
`the projects I (John Dallard) will work on for the first half of my Google STEP internship`

Reply:
I understand. I'll make that quick change!